pkg/ndctl/ndctl.go

```diff
@@ -104,7 +106,7 @@ func (ctx *Context) GetNamespaceByName(name string) (*Namespace, error) {
 			}
 		}
 	}
-	return nil, fmt.Errorf("Not found")
+	return nil, errors.Wrapf(os.ErrNotExist, "namespace '%s' not found", name)
```

Comment:
Sorry, still not convinced that reusing `os.ErrNotExist` is the right approach here. It means that the error occurred during some file-level operation, which isn't the case here. What if at some point this layer really wants to expose OS-level errors as well as the semantic "namespace not found"? `os.ErrNotExist` then cannot be used for both.

Please follow the example from https://blog.golang.org/go1.13-errors and create your own `NotFound` sentinel value.

Also consider what the final error text will look like when wrapping: the code above leads to "namespace 'foo' not found: file does not exist" (https://golang.org/src/internal/oserror/errors.go). "file" - which file?!

That message doesn't make sense, which shows that reusing `os.ErrNotExist` isn't suitable.

Reply:
Defined new error codes for DeviceManager interface: https://github.com/intel/pmem-csi/pull/383/commits/fe4f6e93e200bb7360244e9fa301f6b63911ad14

Reply:
This change also demands go.1.13 in CI to build.

Reply:
Why does this need Go 1.13? You don't need to use the new features from Go 1.13; github.com/pkg/errors is good enough. With "follow the example rom https://blog.golang.org/go1.13-errors and create your own NotFound sentinel value" I meant this:
```
var ErrNotFound = errors.New("not found")
```
That works also in older Go.

Reply:
I would have preferred to switch from Go 1.12 to 1.13 in a separate PR. Now I am not sure how to proceed: rewrite the code so that it works with 1.12 or switch to 1.13 as minimum version? Suggestions?

Reply:
> I would have preferred to switch from Go 1.12 to 1.13 in a separate PR. Now I am not sure how to proceed: rewrite the code so that it works with 1.12 or switch to 1.13 as minimum version? Suggestions?


Ok, then shall we deal this as part of #420, which have other changes to device manager and even unit tests. In this PR I would prefer ephemeral code to make use of existing `DeviceManager` API do a `_nil_` check for `GetDevice()` call in NodeServer, as it was in my initial change?

Reply:
> In this PR I would prefer ephemeral code to make use of existing DeviceManager API

Okay.

Reply:
Now I reverted the error code change. Now it's ready for final review.